internal/signer/darwin/signer.go

```diff
@@ -20,6 +20,8 @@ import (
 	"signer/keychain"
 	"signer/util"
 	"time"
+
+	"github.com/googleapis/enterprise-certificate-proxy/utils"
```

Comment:
Hmm, I don't like introducing this go mod dependency here. For one thing, it will break the copybara sync to google3 (unless we add further copybara transforms in the config) and kokoro transforms. For now, it may be better to duplicate logging util code into the respective signer/util sub directories to avoid this dependency. Go mod and vendoring is excruciating painful to get right, and made worse by google3 import.